ale_linters/haskell/ghc.vim

```diff
@@ -5,69 +5,18 @@ if exists('g:loaded_ale_linters_haskell_ghc')
     finish
 endif
```

Comment:
Aha. That's probably why `ale#utils#Reset()` in an after block of the vader tests didn't work